arch/cpu/cc26x0-cc13x0/rf-core/prop-mode.c

```diff
@@ -1057,10 +1061,17 @@ on(void)
     rf_core_set_modesel();
 
     /* Apply patches to radio core */
-    rf_patch_cpe_genfsk();
+    if (RF_prop.cpePatchFxn != 0)
+        RF_prop.cpePatchFxn( );
+
+    if (RF_prop.mcePatchFxn != 0)
+        RF_prop.mcePatchFxn( );
+
```

Comment:
We'll need to do something more clever here and somehow use a single call/macro that can select and apply the correct patches.